### DISCUSSION TITLE
ERRO AO SALVAR ARQUIVO

### DISCUSSION THREAD

**Discussion by Michelals1980** [dgadelha/Portugol-Webstudio#286] [Dúvidas / Perguntas]
NÃO ESTOU CONSEGUINDO SALVAR NO COMPUTADOR O ARQUIVO CRIADO COM OS CODIGOS EM PORTUGOL. ALGUEM PODER ME AJUDAR?

**Comment by meropimentel** [comment]
Eu também sofrendo muito com essa plataforma. Demorando mais com os problemas do que com o treinamento.

Muitas vezes não grava, não renomeia as abas, para o auto complemento dos comandos (escolha, leia...)e assim vai os problemas.... 
Minha opinião "SEM CONDIÇÕES PARA TRABALHO E PARA APRENDIZAGEM"
Acostumado com o PyCharm e VS Code, que nunca deu um pro comigo.
Tentei fechar o curso de LÓGICA DE PROGRAMAÇÃO em um dia, mas não foi possível devido aos problemas que a cada nova etapa aparece algo para emperrar. Já passando do segundo dia.

**Reply by dgadelha** [reply]
Você consegue gravar a tela com os problemas acontecendo e colocar aqui?

Se possível, incluir a versão do navegador que você está usando.
